lib/src/model/movies_result.dart

```diff
@@ -0,0 +1,70 @@
+class MoviesResult {
+  int? voteCount;
+  late int id;
+  bool? video;
+  num? voteAverage;
+  String? title;
+  double? popularity;
+  late String posterPath;
+  String? originalLanguage;
+  String? originalTitle;
+  List<int> genreIds;
+  String? backdropPath;
+  bool? adult;
+  String? overview;
+  String? releaseDate;
+
+  MoviesResult({
+    this.voteCount,
+    required this.id,
+    this.video,
+    this.voteAverage,
+    this.title,
+    this.popularity,
+    required this.posterPath,
+    this.originalLanguage,
+    this.originalTitle,
+    required this.genreIds,
+    this.backdropPath,
+    this.adult,
+    this.overview,
+    this.releaseDate,
+  });
```

Comment:
Why id, posterPath and genreIds are required and the other parameters dont?

Reply:
Because i'm sure i'm going to use them, so I ensured they wont be null. I think I should add some more (title, vote average for example) for when I do the DetailsPage but i didnt dig too deep yet. Should I set everyone as required?

Reply:
No it's okay, only add required if you are sure that you are going to received or used it always.